sql/2024/cms/top_cms_by_geo.sql

```diff
@@ -0,0 +1,68 @@
+#standardSQL
```

Comment:
In testing this I'm getting "There is no data to display." as an error.

I don't think it was updated correctly.  Maybe the dateformat is wrong?

Reply:
I do not get this error. Can you confirm it is still an error please?